t/10-jobs.t

```diff
@@ -695,6 +697,46 @@ subtest 'carry over, including soft-fails' => sub {
             like($notes->{hook_result}, qr/delayed/, 'real hook cmd from config called if result matches (5)');
             is $notes->{hook_rc}, 142, 'exit code of the hook cmd is as expected';
             is $job_info->{retries}, 1, 'hook script has been retried once because of delay';
+
+            subtest "Retry hook script with linear backoff..." => sub {
```

Comment:
This high level of indentation makes it hard to read. Please move this subtest out onto the global scope